src/openshift/cluster.ts

```diff
@@ -71,12 +72,14 @@ export class Cluster extends OpenShiftItem {
     static async login(): Promise<string> {
         const response = await Cluster.requestLoginConfirmation();
         if (response !== 'Yes') return null;
-        const loginMethod = await window.showQuickPick(['Credentials', 'Token'], {placeHolder: 'Select the way to log in to the cluster.'});
+        const loginMethod = await window.showQuickPick(['Credentials', 'Token', 'ocLogin'], {placeHolder: 'Select the way to log in to the cluster.'});
```

Comment:
I think we should have a more explicit label here

Reply:
... and it is not ocLogin, it's still the same token login, we just reading some of the data, cluster url and token, form clipboard.

Reply:
@dgolovin and @jeffmaury  should I keep only token login option only (instead of oclogin) and if a token is detected from clipboard then I will show Information message that token is detected if user select `yes` then it will automatically login and if user select `No` then I will redirect to manual login throw url and token(https://github.com/redhat-developer/vscode-openshift-tools/blob/master/src/openshift/cluster.ts#L130)